sample/EasyTables/CSharp/QueueToInItemToOutItem/run.csx

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json.Linq;
+
+public static void Run(string input, Item queriedItem, out Item newItem)
+{
+    newItem = new Item();    
```

Comment:
These samples were for me to test that things worked as expected; They're getting cleaned up today. This one actually found a bug in the generated IL.

I can move this to an E2E test to make sure it doesn't regress.


---
In reply to: [56188825](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56188825) [](ancestors = 56188825)